packages/ocamlformat_support/ocamlformat_support.0.1/opam

```diff
@@ -0,0 +1,13 @@
+opam-version: "1.2"
+maintainer: "OCamlFormat Team <ocamlformat-team@fb.com>"
+authors: "Pierre Weis"
+homepage: "https://github.com/ocaml-ppx/ocamlformat/tree/support"
+bug-reports: "https://github.com/ocaml-ppx/ocamlformat/issues"
```

Comment:
if you could add a dev-repo field here as well, then pinning with `opam --dev` will be easier for users.